.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+<!-- Thank you for your pull request! Please provide a description of the changes in this PR in the field above and provide the following information. -->
+
+Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR.
+
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Necessary documentation (if appropriate) have been added / updated
+- [ ] DCO signoffs have been added to all commits, including this PR
+
+#### PR type
+What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
+- [ ] Bugfix <!-- non-breaking change which fixes an issue -->
+- [ ] Feature <!-- non-breaking change which adds functionality -->
+- [ ] Code style update (formatting, local variables)
+- [ ] Refactoring (no functional changes, no API changes)
+- [ ] Build related changes
+- [ ] CI related changes
```

Comment:
Do we need `Code style update` , `Build related changes` and `CI related changes` ? in the PR ?   For an end user I am not sure of the value of these being called out.

Reply:
> Do we need `Code style update` , `` Build related changes``Refactoring (no functional... `` and `CI related changes` ? in the PR ? For an end user I am not sure of the value of these being called out. If anyone wanted to get this information they could do a compare of the files in the PR to determine and having these questions asked may be too much information to capture.

@Joe-Winchester Done. Removed the extra options.